site/topics/booleans/booleans.rst

```diff
@@ -1,6 +1,59 @@
-*******************************
-Logic, Booleans, More Functions
-*******************************
+***************
+Predicate Logic
+***************
+
+Logic First
+===========
+
+* Sometimes things are ``True`` or ``False``
+* I'm betting you actually are pretty familiar with some LOGIC
+    * AND
+    * OR
+    * NOT
+
+.. admonition:: Activity
+
+    Write out the truth tables for the logical operations **AND**, **OR** and **NOT**.
+
+    Don't know what a 'logical operator' or 'truth table' is? No problem, Ask:
+
+    * Wikipedia
+    * Google
+    * Your neighbour
+    * Just don't ask me!
```

Comment:
Could maybe use some toning down.

Reply:
I'll address these when I go for the actual rework of the content. This PR is just moving things around.

Reply:
Ack.